Softeq.XToolkit.Permissions.Droid/SdkVersion.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using Android.OS;
+using Microsoft.Maui.ApplicationModel;
+
+namespace Softeq.XToolkit.Permissions.Droid
+{
+    internal static class SdkVersion
+    {
+        public static bool IsBuildVersionLower(BuildVersionCodes versionCode)
+        {
+            return Platform.AppContext.ApplicationInfo?.TargetSdkVersion <= versionCode;
+        }
+
+        public static bool IsBuildVersionAtLeast(BuildVersionCodes versionCode)
+        {
+            return Platform.AppContext.ApplicationInfo?.TargetSdkVersion >= versionCode;
+        }
+
+        public static bool IsDeviceVersionAtLeast(BuildVersionCodes versionCode)
```

Comment:
Namings look misleading. I would say that `IsBuildVersionAtLeast` should be called `IsTargetSdkVersionAtLeast` and `IsDeviceVersionAtLeast` should be called `IsAndroidVersionAtLeast`. Do you know why it's called this way?

Reply:
Also I wonder whether we should use `OperatingSystem.IsAndroidVersionAtLeast` everywhere now instead of these methods

Reply:
@pavel-leonenko 
`IsBuildVersionLower` => `IsTargetSdkLower`
`IsBuildVersionAtLeast` => `IsTargetSdkAtLeast`
`IsDeviceVersionAtLeast` => `IsBuildSdkAtLeast`